src/navigator/Navigator.ts

```diff
@@ -29,6 +29,8 @@ interface Navigator {
 
   startReadAloud?(): void;
 
+  startReadAloudBySegment?(startTime: number, endTime: number): void;
```

Comment:
If possible to avoid confusion when passing prop for this function, its better to prop an object prop

```suggestion
  startReadAloudBySegment?( { startTime, endTime }: { startTime: number, endTime: number }): void;
```